tests/unit/serviceSpec.js

```diff
@@ -74,5 +74,21 @@ describe('service', function () {
             });
             $httpBackend.flush();
         }));
+
+        it('should freeze bundle', inject(function (locale, $httpBackend) {
+            var resp = 'lang file not found';
+            $httpBackend.expectGET('languages/en-US/common.lang.json').respond(200, {
+                "yes": "Yes"
+            });
+            locale.$$bundleReady('common').then(function (bundle) {
```

Comment:
Does this still run? The test should fail as $$bundleReady no longer available. Also, need to change it to check the event.